### ISSUE TITLE
[Feature] Change Language in Welcome screen

### ISSUE THREAD

**Issue by rolznz** [getAlby/lightning-browser-extension#1869]
### Feature description

In the welcome screen I don't see any way to change the extension language. I also tried changing my language from my chrome settings to Spanish, then going back to the extension tab and refreshing, but the welcome page was still in English. Trying to open the extension options will just re-open the welcome screen since the setup hasn't been completed yet.

### Describe the solution

Either the language should update when you change your language in the browser settings, or there should be a way to change the language in the welcome screen (like a dropdown in the top right corner)

**Comment by escapedcat** [comment]
Hm, when this was built this worked I believe.  
This should  take of of using the correct language: https://github.com/getAlby/lightning-browser-extension/blob/master/src/i18n/i18nConfig.ts#L60  
If this doesn't work (anymore) this is a bug.

**Comment by lujakob** [comment]
@rolznz @escapedcat I debugged this: the applied `LanguageDetector` in `i18nConfig.ts` works as expected and switches to the browser settings preferred language (I tested in Chrome and it uses the first in list). The problem is, in SettingsContext we listen for the `settings.locale` changes (see https://github.com/getAlby/lightning-browser-extension/blob/master/src/app/context/SettingsContext.tsx#L132) and update the language accordingly, back to English, in case the user selected English before in the settings page. 

So I guess we have to choose our priority: either just detect the browser language on page load (app start) or explicitly let the user select (and store) the language on settings page. I guess this has more priority, as it is implemented. In this case I would leave as is (let the browser detection do its work and then the SettingsContext useEffect set `settings.local`) and just add a language selector to the welcome page. 

In case a user installs for the first time and never manually selected a language in the settings before, I guess the i18nConfig detected language is prioritized.

**Comment by lujakob** [comment]
In case you agree I could implement it. Question left would be how to style it? Placing language selector at the top right corner could be an option? Just not sure if the select style from the settings page is ideal? Maybe anyone else has suggestions..

**Comment by escapedcat** [comment]
I guess it would make sense to offer a language-select as well. tbh we could remove the [browser language detection](https://github.com/getAlby/lightning-browser-extension/blob/master/package.json#L54) as well then.

Did some digging:

We had a select. It was introduced here:
https://github.com/getAlby/lightning-browser-extension/pull/711/files#diff-784da3b6df7bd2be24bdb6e8b8e068eb2760d9f773695e6c2b431729fda0971c
Initally we only showed this because we didn't have any translations back then. The plan was to enabel this, once we do have translations.

Then it was removed here:
https://github.com/getAlby/lightning-browser-extension/pull/1397/files#diff-784da3b6df7bd2be24bdb6e8b8e068eb2760d9f773695e6c2b431729fda0971cL132-L139
It looks like we didn't do this on purpose though. The change is not mentioned in the PR description nor in the commit-messages.
@bumi you remember if this was done on purpose? If not we could just re-add this, right?
